docs/repo/Getting Started.md

```diff
@@ -17,14 +17,14 @@ This builds, deploys and run tests.
 
 ### Debugging/Deploying
 
+By default when you build inside Visual Studio or the command-line, the project system and other binaries gets deployed to the _RoslynDev_ experimental instance of Visual Studio. They are setup so that they _override_ any binaries that come with Visual Studio.
 
 #### Visual Studio
-By default when you build, the project system and other binaries gets deployed to the _RoslynDev_ experimental instance of Visual Studio. They are setup so that they _override_ any binaries that come with Visual Studio.
 
 To start debugging:
 
 1. Open __src\ProjectSystem.sln__
-2. Right-click on the __ProjectSystemSetup__ project, and choose __Set As Startup Project__
+2. Right-click on the __ProjectSystemDogfoodSetup__ project, and choose __Set As Startup Project__
```

Comment:
Why do we even have `ProjectSystemDogfoodSetup`? I've always run `ProjectSystemSetup`.